apple-silicon-support/modules/default.nix

```diff
@@ -21,6 +21,8 @@
             localSystem.system = cfg.pkgsSystem;
             overlays = [ cfg.overlay ];
           }
+        else if ! pkgs ? mesa-asahi-edge then
+          pkgs.extend cfg.overlay
```

Comment:
I'm happy to drop this part if you don't want to support people passing in `pkgs` directly.

Reply:
Two concerns:
* Not comfortable supporting something which breaks `nixpkgs.overlays`, that really should work
* This method of detecting that an un-overlayed set has been passed in seems fragile.

I would like this dropped.

Reply:
Removed :+1: